rest_framework/renderers.py

```diff
@@ -790,3 +791,17 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
                     "test case." % key
                 )
         return encode_multipart(self.BOUNDARY, data)
+
+
+class CoreJSONRenderer(BaseRenderer):
```

Comment:
There was a move in 3.1 to remove features which had third-party dependencies from the core and put them into third-party packages that were still highly visible. See [django-rest-framework-xml](https://github.com/jpadilla/django-rest-framework-xml), [django-rest-framework-yaml](https://github.com/jpadilla/django-rest-framework-yaml), and [django-rest-framework-oauth](https://github.com/jpadilla/django-rest-framework-oauth) for examples.

Is that still something we are pushing for?